tests/Fixer/FunctionNotation/NativeFunctionInvocationFixerTest.php

```diff
@@ -564,6 +564,38 @@ public function & strlen($name) {
         ];
     }
 
+    /**
+     * @dataProvider provideFixWithImportedFunctionCases
+     *
+     * @param string      $expected
+     * @param null|string $input
+     */
+    public function testFixImportedFunction($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    /**
+     * @return array
+     */
+    public function provideFixWithImportedFunctionCases()
+    {
+        return [
```

Comment:
Those test cases don't prove the bug you are fixing: by default, the fixer only adds the leading `\` to native PHP functions, which `bar()` and `baz()` aren't. You should configure the fixer before calling `doTest()`:
```php
$this->fixer->configure([
    'include' => ['@all'],
]);
```
I would also add a test case with a native PHP function such as `json_decode()`.